--- ./src/smtube.pro.orig	2020-06-19 12:02:59.047658202 +0100
+++ ./src/smtube.pro	2020-06-19 12:04:57.370202021 +0100
@@ -1,5 +1,6 @@
 TEMPLATE = app
 QT += network
+INCLUDEPATH += qtsingleapplication
 
 CONFIG += release
 #CONFIG += debug
@@ -9,6 +10,7 @@ DEFINES += USE_PLAYERS
 DEFINES += STYLE_SWITCHING
 DEFINES += CODEDOWNLOADER
 DEFINES += USE_SITES
+DEFINES += USE_SINGLE_APPLICATION
 
 # If Qt >= 5.4
 greaterThan(QT_MAJOR_VERSION, 4):greaterThan(QT_MINOR_VERSION, 3) {
